### PR TITLE
Clarified support & fixed synopsis for global mapping

### PR DESCRIPTION
**Pull request by nedpyle**
# PR Summary

Fixed the synopsis and updated the supported and unsupported scenarios for use of New-SmbGlobalMapping

## PR Checklist

- [X] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [X] **Summary:** This PR's summary describes the scope and intent of the change.
- [X] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [X] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: docset/winserver2022-ps/smbshare/New-SmbGlobalMapping.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ New-SmbGlobalMapping [[-LocalPath] <String>] [-RemotePath] <String> -Credential
 ```
 
 ## DESCRIPTION
-The **New-SmbGlobalMapping** cmdlet creates an SMB global mapping on the SMB client to an SMB share. Global mappings allow all users to use the same mapping. Its primary use is for Windows Containers.
+The **New-SmbGlobalMapping** cmdlet creates an SMB global mapping on the SMB client to an SMB share. Global mappings allow all users to use the same mapping. Its primary use is for Windows Containers. Global mappings support standalone and failover cluster SMB shares, they do not support DFS Namspace folder shares.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/smbshare/New-SmbGlobalMapping.md:26]:
```suggestion
The **New-SmbGlobalMapping** cmdlet creates an SMB global mapping on the SMB client to an SMB share. Global mappings allow all users to use the same mapping. Its primary use is for Windows Containers. Global mappings support standalone and failover cluster SMB shares, they do not support DFS Namespace folder shares.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @nedpyle, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by nedpyle** [comment]
@JohanFreelancer9 do I need to do anything else here for this to be published?

**Comment by JohanFreelancer9** [comment]
@nedpyle, nothing more is required from you at this time. @scanum, could you please help to get this merged. Thanks.

**Review by dariomws — APPROVED** [review]
I think this cmdlet is also available in Windows Server 2019 and 2016.

**Review by get-itips — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**